.github/linters/.eslintrc.yml

```diff
@@ -78,5 +77,9 @@ rules:
     '@typescript-eslint/restrict-plus-operands': 'error',
     '@typescript-eslint/space-before-function-paren': 'off',
     '@typescript-eslint/type-annotation-spacing': 'error',
-    '@typescript-eslint/unbound-method': 'error'
+    '@typescript-eslint/unbound-method': 'error',
+
+    # Fixes enum linting
+    'no-shadow': 'off',
+    '@typescript-eslint/no-shadow': 'error'
```

Comment:
A false positive was happening when I was declaring the TestStatus enum

https://github.com/eslint/eslint/issues/13644

Reply:
as we are not using the `enum` anymore, this is not necessary. I removed it